src/osal/lv_freertos.c

```diff
@@ -396,9 +423,12 @@ static void prvCheckMutexInit(lv_mutex_t * pxMutex)
     }
 }
 
-#if !USE_FREERTOS_TASK_NOTIFY
 static void prvCondInit(lv_thread_sync_t * pxCond)
 {
+#if USE_FREERTOS_TASK_NOTIFY
+    pxCond->xTaskToNotify = NULL;
```

Comment:
Implement this, similar as it is done for semaphores.
I see it like:
```
    pxCond->xTaskToNotify = xTaskGetCurrentTaskHandle();
    pxCond->xIsInitialized = pdTRUE;
    pxCond->xSyncSignal = pdFALSE;
```
You might have the last two common for semaphores and sygnal thread.

Reply:
I mean to replace all the code under `#if USE_FREERTOS_TASK_NOTIFY` with the above suggestion and see if it works to you.